ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/Model.hs

```diff
@@ -392,22 +401,17 @@ addBlock :: forall blk. LedgerSupportsProtocol blk
          => TopLevelConfig blk
          -> blk
          -> Model blk -> Model blk
-addBlock cfg blk m = Model {
-      volatileDbBlocks = volatileDbBlocks'
-    , immutableDbChain = immutableDbChain m
-    , cps              = CPS.switchFork newChain (cps m)
-    , currentLedger    = newLedger
-    , initLedger       = initLedger m
-    , iterators        = iterators  m
-    , valid            = valid'
-    , invalid          = invalid'
-    , currentSlot      = currentSlot  m
-    , maxClockSkew     = maxClockSkew m
-    , isOpen           = True
-    }
+addBlock cfg blk m =
```

Comment:
Would a pattern match on `ignoreBlock` be more readable than an `if`?

Reply:
I condensed this code a bit :+1:
```diff
diff --git a/ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/Model.hs b/ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/Model.hs
index 2c8f5c7eef..05f7cabe74 100644
--- a/ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/Model.hs
+++ b/ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/Model.hs
@@ -402,14 +402,10 @@ addBlock :: forall blk. LedgerSupportsProtocol blk
          -> blk
          -> Model blk -> Model blk
 addBlock cfg blk m =
-  if ignoreBlock
-    then m
-    else
-      chainSelection cfg $
-        m
-          { volatileDbBlocks =
-              Map.insert (blockHash blk) blk (volatileDbBlocks m)
-          }
+  | ignoreBlock = m
+  | otherwise   = chainSelection cfg m {
+        volatileDbBlocks = Map.insert (blockHash blk) blk (volatileDbBlocks m)
+      }
   where
     secParam = configSecurityParam cfg
     immBlockNo = immutableBlockNo secParam m
```